tensorflow/python/eager/def_function.py

```diff
@@ -1406,6 +1415,8 @@ def function(func=None,
     experimental_compile: If True, the function is always compiled by
       [XLA](https://www.tensorflow.org/xla). XLA may be more efficient in some
       cases (e.g. TPU, XLA_GPU, dense tensor computations).
+    experimental_type_tracing: When true, arguments type annotated with
```

Comment:
To avoid duplication, let's only keep the docstring for the exported symbol, and in all other instances write just "See <module>.<function>".